setup.py

```diff
@@ -47,6 +47,9 @@ def package_files(directory):
     "pytz",
     "requests>=2.17.3",
     "packaging",
+    # 3.7.0 or newer is required to use `importlib_metadata.packages_distributions`:
```

Comment:
Can we add an explanation about why we need to use `packages_distributions`?

Reply:
Added the explanation!